install_config/aggregate_logging.adoc

```diff
@@ -440,8 +440,19 @@ endif::openshift-enterprise[]
 [[scaling-elasticsearch]]
 *Changing the Scale of Elasticsearch*
 
-To scale Elasticsearch up, create more deployments. The deployer provides
-a template to create Elasticsearch deployments:
+If you need to scale up the number of Elasticsearch instances your cluster uses
+it is not as simple as changing the number of replicas. This is due to the nature
+of Persistent volumes and how Elasticsearch is configured to store its data.
+
+Instead, you will need to create a Deployment Configuration for each replica.
+
+The Deployer created templates during installation with the Elasticsearch
+configurations provided to it: `logging-es-template` and `logging-es-ops-template`
+if the Deployer was run with `ENABLE_OPS_CLUSTER=true`.
+
+To create an additional Deployment Configuration, run the following command against
+the Elasticsearch cluster you'd like to scale up for (`logging-es-template` or
+`logging-es-ops-template`).
```

Comment:
@ewolinetz I'm at a loss for how or where to say this, but they should really adjust some parameters in the template before adding instances. The quorum should be 1 + size/2 and the recover_after / expected nodes should be the size of the cluster. And after those exist, they should go back to the existing DCs and make the same changes. Actually they should scale everything down first, make the changes / additions, then scale everything up.